src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Extensions/OpenApiSchemaExtensions.cs

```diff
@@ -382,5 +384,120 @@ public static bool IsOpenApiSchemaDictionary(this OpenApiSchema schema)
 
             return false;
         }
+
+        /// <summary>
+        /// Applying <see cref="ValidationAttribute"/> to <see cref="OpenApiSchema"/>
+        /// </summary>
+        /// <param name="schema"><see cref="OpenApiSchema"/> instance.</param>
+        /// <param name="customAttributes">DataAnnotation attributes</param>
+        public static void ApplyValidationAttributes(this OpenApiSchema schema, IEnumerable<ValidationAttribute> customAttributes)
```

Comment:
Can you please make sure your changes are verified by test cases on:

https://github.com/Azure/azure-functions-openapi-extension/blob/main/test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Extensions/OpenApiSchemaExtensionsTests.cs

Reply:
Done